### ISSUE TITLE
Exception when closing one of several Gauge projects open

### ISSUE THREAD

**Issue by j-morsum** [getgauge/Intellij-Plugin#384]
### Expected behavior
No exception should be thrown.

### Actual behavior
I often have more than one Gauge projects open so when I close one I see an error on the open project. On some occasions I see the same exception while opening a project. 

There isn't any message in the log files. The exception shown is:
```
Exception occurred in Gauge plugin
Please log an issue here with the following details and reload the project.
	
Gauge daemon initialized and listening on port: 54920	
	* Idea version: 2019.1.3
	* API version: IC-191.7479.19
	* Plugin version: 0.3.16.nightly-2019-05-15
	* Gauge version: 1.0.5
```
### Steps to reproduce
1. Open two or more Gauge projects on Intellij;
2. Close one of the projects (either by clicking on the X button or going to "File" > "Close Project")

### Version information
* IntelliJ version and edition(Community/Ultimate):  2019.1.3 Community
* OS information: Windows 10 64 bits
* Gauge IntelliJ plugin version: 0.3.16.nightly-2019-05-15
* Does the project uses any Build tool(Gradle/Maven)? No
```
Gauge version: 1.0.5
Commit Hash: 562f036

Plugins
-------
html-report (4.0.8)
java (0.7.2)
json-report (0.3.1)
screenshot (0.0.1)
spectacle (0.1.3)
xml-report (0.2.1)
```

**Comment by nehashri** [comment]
@j-morsum I am able to replicate this issue in windows, we will look at fixing this soon.

**Comment by Debashis9012** [comment]
This issue has been fixed with the latest nightly version: ```0.3.16.nightly-2019-07-09```
